src/content/getting-started/quickstart/kind/_index.md

```diff
@@ -23,12 +23,14 @@ To create kind clusters and deploy Submariner, run:
 ```bash
 git clone https://github.com/submariner-io/submariner-operator
 cd submariner-operator
-make deploy
+make deploy using=lighthouse
 ```
 
 By default, the automation configuration in the submariner-io/submariner-operator repository deploys two clusters, with cluster1 configured as
 the Broker. See the [settings](https://github.com/submariner-io/submariner-operator/blob/devel/.shipyard.e2e.yml) file for details.
 
+Refer to [`Submariner Deployment Customization`](../../../development/shipyard/advanced/) for examples on how to customize the environment outside of defaults values.
```

Comment:
I'm not sure we should direct users from the quickstart to these advance Shipyard reference docs for developers. Users should not need to know those Shipyard details, only devs doing work with Shipyard. For more user-facing docs about using custom Shipyard envs, they should see https://submariner.io/development/building-testing/.

Reply:
I agree it could be a lot, but for something simple like deploying with a different cable-driver, I think it is useful. When you are new, you don't know what to search for, so I was hunting and pecking for a day trying to figure out why lighthouse wasn't being deployed (but was working on Manual Steps). Link both?

Reply:
It's great to have this fresh view and I want to act on this feedback, but I think we shouldn't ask users to dig through the Shipyard Advanced Options to learn something basic like how to set the cable driver. We have more user-focused docs for that. We want people to use the `using` flags, not manually pass cluster/deploy options. Maybe we should make this section stand out more somehow, link to it?

> The cable driver used to connect clusters can also be selected with using flags:
make e2e using=vxlan

https://submariner.io/development/building-testing/#end-to-end-tests

Reply:
This updated version you have with both is okay with me. :+1: